lib/internal/url.js

```diff
@@ -896,6 +896,81 @@ defineIDLClass(URLSearchParams.prototype, 'URLSearchParams', {
     update(this[context], this);
   },
 
+  sort() {
+    const a = this[searchParams];
+    const len = a.length;
+    if (len <= 2) {
+      return;
+    }
+
+    // arbitrary number found through testing
+    if (len < 118) {
```

Comment:
We should look to benchmark this on a number of different systems to see if this number holds up.